compiler/ast/zed/value.go

```diff
@@ -1,5 +1,10 @@
 package zed
 
+type Node interface {
```

Comment:
Nit: I think types.go would be a better place for this since it's embedded in `Type`.

Reply:
Actually don't need this at all in astzed so I got rid of it.